sdk/protocol-plugins/tsconfig.json

```diff
@@ -5,6 +5,11 @@
     "outDir": "dist",
     "baseUrl": "."
   },
-  "include": ["src/**/*.ts", "tests/**/*.ts", "tests/mocks/*.ts"],
+  "include": [
+    "src/**/*.ts",
+    "tests/**/*.ts",
+    "tests/mocks/*.ts",
+    "../protocol-plugins-common/src/actions/SkippedAction.ts"
```

Comment:
is this one needed here? It looks suspicious

Reply:
Indeed it was wrong. Good catch!